components/src/dynamo/trtllm/llm_engine.py

```
@@ -268,7 +284,31 @@ async def start(self, worker_id: int) -> EngineConfig:
+            # the endpoint down, metrics populate the global Prometheus REGISTRY
+            # but are not served over HTTP in the unified path.  The legacy
+            # workers/llm_worker.py path has this wired correctly at line 563.
+            try:
+                self._metrics_collector = MetricsCollector(
+                    {
+                        "model_name": self.served_model_name or self.model_name,
+                        "engine_type": "trtllm",
+                    }
+                )
+                logger.info("TensorRT-LLM MetricsCollector initialized")
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)